Examples/Python/python/acts/examples/simulation.py

```diff
@@ -378,8 +427,6 @@ def addFatras(
         the output folder for the Csv output, None triggers no output
     outputDirRoot : Path|str, path, None
         the output folder for the Root output, None triggers no output
-    rnd : RandomNumbers, None
-        random number generator
```

Comment:
is there a reason for removing this except that doc == arg?

Reply:
hmm this seems to bie a leftover as well